.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "/home/agreene/.local/share/virtualenvs/operator-courier-8e3EdCOa/bin/python"
```

Comment:
Let's not commit the settings.json for vscode.

Reply:
Thanks for catching that.

Reply:
I use global gitignore for this purpose, makes my life much better.

https://stackoverflow.com/questions/7335420/global-git-ignore